tests/ext/test_updater.py

```diff
@@ -755,19 +792,75 @@ async def set_webhook(*args, **kwargs):
                 assert self.message_count == 0
 
             # We call the same logic twice to make sure that restarting the updater works as well
-            await updater.start_webhook(
-                drop_pending_updates=drop_pending_updates,
-                ip_address=ip,
-                port=port,
-                url_path="TOKEN",
-            )
+            if unix:
+                await updater.start_webhook(
+                    drop_pending_updates=drop_pending_updates,
+                    secret_token=secret_token,
+                    unix=file_path,
+                    webhook_url="string",
+                )
+            else:
+                await updater.start_webhook(
+                    drop_pending_updates=drop_pending_updates,
+                    ip_address=ip,
+                    port=port,
+                    url_path="TOKEN",
+                    secret_token=secret_token,
+                    webhook_url="string",
+                )
             assert updater.running
             update = make_message_update("Webhook")
-            await send_webhook_message(ip, port, update.to_json(), "TOKEN")
+            await send_webhook_message(
+                ip,
+                port,
+                update.to_json(),
+                "" if unix else "TOKEN",
+                secret_token=secret_token,
+                unix=file_path if unix else None,
+            )
             assert (await updater.update_queue.get()).to_dict() == update.to_dict()
             await updater.stop()
             assert not updater.running
 
+    @pytest.mark.skipif(
+        platform.system() == "Windows",
+        reason="Windows doesn't support unix bind",
+    )
+    async def test_unix_webhook_raises(self, updater):
+        async with updater:
+            with pytest.raises(RuntimeError, match="You can not pass unix and listen"):
+                await updater.start_webhook(listen="127.0.0.1", unix="DoesntMatter")
+            with pytest.raises(RuntimeError, match="You can not pass unix and port"):
+                await updater.start_webhook(port=20, unix="DoesntMatter")
+            with pytest.raises(RuntimeError, match="you set unix, you also need to set the URL"):
+                await updater.start_webhook(unix="DoesntMatter")
+
+    @pytest.mark.skipif(
+        platform.system() == "Windows",
+        reason="Windows doesn't support unix bind",
+    )
+    async def test_unix_webhook_path(self, updater, monkeypatch, file_path):
+        async def set_webhook(*args, **kwargs):
+            return True
+
+        monkeypatch.setattr(updater.bot, "set_webhook", set_webhook)
+
+        async with updater:
+            await updater.start_webhook(unix=file_path, webhook_url="string")
+
+            update = make_message_update("Webhook")
+            await send_webhook_message("ip", 123, update.to_json(), unix=file_path)
+            await updater.stop()
```

Comment:
What does this test? :D It's clear to me neither from the test name nor from the content …

Reply:
sorry leftover from me wondering if path works on windows.